Exiled.API/Features/Roles/Scp3114Role.cs

```diff
@@ -235,5 +237,15 @@ public void PlaySound(Scp3114VoiceLines.VoiceLinesName voiceLine = Scp3114VoiceL
         /// <param name="alreadySpawned">The List of Roles already spawned.</param>
         /// <returns>The Spawn Chance.</returns>
         public float GetSpawnChance(List<RoleTypeId> alreadySpawned) => Base is ISpawnableScp spawnableScp ? spawnableScp.GetSpawnChance(alreadySpawned) : 0;
+
+        /// <summary>
+        /// Stops dancing.
+        /// </summary>
+        public void StopDancing() => Dance.SendRpc(x => x.IsDancing = false);
+
+        /// <summary>
+        /// Starts dancing.
+        /// </summary>
+        public void StartDanging() => Dance.SendRpc(x => x.IsDancing = true);
```

Comment:
i just make sure than it's do not get merge in dev like this

Reply:
Fixed [here](https://github.com/Exiled-Team/EXILED/blob/apis-rework/Exiled.API/Features/Roles/Scp3114Role.cs)